packages/mobile/src/app/App.tsx

```diff
@@ -66,6 +67,23 @@ export class App extends React.Component<Props> {
   async componentDidMount() {
     await ValoraAnalytics.init()
 
+    // Handles opening Clevertap deeplinks when app is closed / in background
+    CleverTap.getInitialUrl(async (err: any, url: any) => {
```

Comment:
Looks like CleverTap has types for this: https://github.com/CleverTap/clevertap-react-native/blob/master/index.d.ts#L33
```suggestion
    CleverTap.getInitialUrl(async (err, url) => {
```

Reply:
I was able to get the types for url; however, err did not appear to pickup the types from CleverTap. Fixed in https://github.com/valora-inc/wallet/pull/2012/commits/a19fa197a391f4f86e1a86ea22d3209216401370.